### ISSUE TITLE
Spike: How can we target US first-time visitors?

### ISSUE THREAD

**Issue by shesah** [guardian/dotcom-rendering#12187]
For our testing of the US branding banner, we'd like to target first time/infrequent visitors to the Guardian US. For this ticket, we would like to a) is this possible? b) how complex is it?

**Comment by domlander** [comment]
https://chat.google.com/room/AAAAFdv9gK8/Vw8O2b-IUwg/Vw8O2b-IUwg

**Comment by domlander** [comment]
The page doesn't refresh when the user initially accepts/rejects cookies. It only refreshes when the user updates their consent status.

**Comment by domlander** [comment]
Data privacy (Shaun McMullen) is going to re-review the above question with the DPO

**Comment by shesah** [comment]
Can we use localstorage to store a banner dismissal for reject all users? WebX would like to know whether we can target some US-based reject-all users with a banner that will show on each page view until they dismiss it. It appears we are currently already doing this with the "rejection hurts..." banner; the key in localstorage for this is gu.prefs.engagementBannerLastClosedAt.

**Comment by domlander** [comment]
Shervin and I met with Shaun McMullen yesterday afternoon. He is going to check with Paul Brown either today or Friday. Shaun is on holiday next week.

**Comment by domlander** [comment]
Reply from Shaun: https://groups.google.com/a/guardian.co.uk/g/dotcom.platform/c/RU4mjMFChoU

TLDR: We can present users in the US with banners asking them to register and highlighting the Guardian's journalism. We can remember a user's choice to dismiss a screen, however we would normally do this and would not need to rely on a user's CMP consent choice in their local storage.
